Unix/tests/codec/mof/blue/test_lex.cpp

```diff
@@ -28,20 +28,20 @@
 
 #define TEST_ASSERT(x) NitsAssert(x, MI_T(""))
 
-#if (defined(sun) || defined(hpux) || defined(aix)) && defined(CONFIG_ENABLE_WCHAR)
+#if defined(sun) || defined(hpux) || defined(aix)
 
 // 
 // This failes due to difficulty supporting four byte character sets in codec/mof/parser/utility.c
 // An issue has been filed.
 //
-#define DISABLE_WCHAR_TESTS 1
+#define DISABLE_CHAR_TESTS 1
 #endif
```

Comment:
Why define a new variable?

Reply:
In line 44 of file test_lex.cpp, the DISABLE_CHAR_TESTS are never defined.  I think there
 is typo in line 37, it should be DISABLE_CHAR_TESTS, but not DISABLE_WCHAR_TESTS.